src/test/basics/RangeSet_test.cpp

```diff
@@ -26,69 +26,132 @@ namespace ripple {
 class RangeSet_test : public beast::unit_test::suite
 {
 public:
-    RangeSet createPredefinedSet ()
+    void
+    testSetAndHas()
```

Comment:
these are good tests - the only other thing I'd like to see is `setRange` with values that overlap and the front/back/middle of an existing range. If that's already there and I missed it, then 👍